changelog.d/1157.doc.rst

```diff
@@ -0,0 +1 @@
+Install with docs and test feature in development mode
```

Comment:
```suggestion
Add installing docs and test dependencies when contributing
```

maybe something like this? the current message makes it feel like Subliminal is installing some stuff on its own

Reply:
Thanks!